src/bors/handlers/pr_events.rs

```diff
@@ -466,7 +466,9 @@ mod tests {
                     pr.pr_status == PullRequestStatus::Open
                 })
                 .await?;
-            tester.merge_pr(default_repo_name(), pr.number.0).await?;
+            tester
+                .merge_pr(default_repo_name(), pr.number.0, true)
```

Comment:
I missed this function, let's also remove `send_webhook: bool,` from its parameters, now that we have a better way of blocking webhooks.